pages/community/election.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: page
+title: US-RSE Steering Committee Elections
+subtitle:
+bigimg:
+permalink: /election/
+---
+
+The 2021 US-RSE Steering Committee elections are happening this December.
+
+## Timeline
+
+The timeline for this year's elections will be posted soon.
+
+
+## Positions and Eligibility
+Five Steering Committee seats are open in this election, each for a 2-year term (January 1, 2022 through December 31, 2023). The following steering committee members have terms that end this year: Jeff Carver, Ian Cosden, Charles Ferenbaugh, Sandra Gesing, and Christina Maimone. All US-RSE members in good standing as of the day nominations open are eligible for nomination.  
+
+A membership verification email will be sent to all members in October. The message will be sent directly from leadership@us-rse.org, not through the Google group used for most email communications.   Individuals who do not receive the membership verification email may join the organization by filling out the membership form at [https://us-rse.org/join/](https://us-rse.org/join/).  For questions or help with membership matters, please contact Election Chairs Julia Damerow and Lance Parsons on Slack or [via email](mailto:jdamerow@asu.edu,lparsons@princeton.edu).
+
+The US-RSE Steering Committee sets its own meeting frequency and timing.  Currently, the Steering Committee is meeting every two weeks.  The Steering Committee meeting frequency and schedule for 2022 will be determined once new members are elected.  
```

Comment:
```suggestion
The US-RSE Steering Committee sets its own meeting frequency and timing.  Currently, the Steering Committee meets every two weeks.  The Steering Committee meeting frequency and schedule for 2022 will be determined once new members are elected.  
```

Reply:
I would actually move this down as a subbullet under 
* Attend and participate in Steering Committee meetings